nvme.c

```diff
@@ -1759,6 +1759,70 @@ static int id_ns(int argc, char **argv, struct command *cmd, struct plugin *plug
 	return err;
 }
 
+static int id_nvmset(int argc, char **argv, struct command *cmd, struct plugin *plugin)
+{
+	const char *desc = "Send an Identify NVM Set List command to the "\
+		"given device, returns entries for NVM Set identifiers greater "\
+		"than or equal to the value specified CDW11.NVMSETID "\
+		"in either binary format or json format";
+	const char *raw_binary = "show infos in binary format";
+	const char *nvmset_id = "NVM Set Identify value";
+	int err, fmt, fd;
+	struct nvme_id_nvmset nvmset;
+
+	struct config {
+		__u16 nvmset_id;
+		int   raw_binary;
+		char *output_format;
+	};
+
+	struct config cfg = {
+		.nvmset_id = 0,
+		.output_format = "normal",
+	};
+
+	const struct argconfig_commandline_options command_line_options[] = {
+		{"nvmset_id",       'i', "NUM", CFG_POSITIVE, &cfg.nvmset_id,       required_argument, nvmset_id},
+		{"raw-binary",      'b', "",    CFG_NONE,     &cfg.raw_binary,      no_argument,       raw_binary},
+		{"output-format",   'o', "FMT", CFG_STRING,   &cfg.output_format,   required_argument, output_format },
```

Comment:
output_format has to bring some help message but i do not see it defined/declared? could you please point out?

for instance, https://github.com/linux-nvme/nvme-cli/blob/d94fda25db81ba5b43b18c05dadff489d50ea0d2/wdc-nvme.c#L1469

Reply:
You can find it at line 155.
`static const char *output_format = "Output format: normal|json|binary";`

It has been declared for a long time as a static value in nvme.c.

Reply:
OK